bin/cylc-cat-log

```diff
@@ -336,47 +313,54 @@ def main():
                 "local tail command template"))
             commands.append(shlex.split(cmd_tmpl % {"filename": filename}))
     elif options.geditor or options.editor:
+        # Copy local or remote job file to a local temp file.
+        viewfile = mkstemp(dir=cylc_tmpdir)[1]
+        if user_at_host:
+            cp = 'scp %s:%s %s' % (user_at_host, filename, viewfile)
+        else:
+            cp = 'cp %s %s' % (filename, viewfile)
```

Comment:
Best to set up the `cp` command as a list at this point? In case any file name contains space characters, etc?